test/plugin/test_cache_strategy.rb

```diff
@@ -0,0 +1,169 @@
+#
+# Fluentd Kubernetes Metadata Filter Plugin - Enrich Fluentd events with
+# Kubernetes metadata
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+require_relative '../helper'
+require 'fluent/plugin/kubernetes_metadata_cache_strategy'
+require 'fluent/plugin/kubernetes_metadata_stats'
```

Comment:
use `require_relative` for these too, to facilitate testing by doing `ADD *.rb /etc/fluent/plugin/` in the image.

Reply:
Updated file to use require_relative and also added to the main class file.
I don't think I need to add any docker statements since this is brought in using the gem file for the plugin.  They will be part of the installation.